backend/src/main/java/com/sidedish/servcie/ProductService.java

```diff
@@ -21,8 +28,28 @@ public List<MainCategoryProductDto> findMainCategory(String mainCategory){
         List<Product> products = productRepository.findAllMainCategoryProduct(mainCategory);
         List<MainCategoryProductDto> dtoList = new ArrayList<>();
         for (Product product : products) {
-            dtoList.add(new MainCategoryProductDto(product.getProductName(), product.getDescription(), product.getPrice(), product.getEventBadge(),product.getMainCategory()));
+            String thumbnail = product.getImage().stream().findFirst().get().getImagePath();
+            dtoList.add(new MainCategoryProductDto(product.getProductName(), product.getDescription(), product.getEarlyDelivery(), product.getPrice(), product.getEventBadge(), product.getEventBadge().getDiscountRate(),product.getMainCategory(), thumbnail));
```

Comment:
여기도 마찬가지입니다. 직접 꺼내서 주입하지 말고, 객체를 넣어서 `MainCategoryProductDto` 가 직접 필요한 원소를 선별해서 스스로 작업할 수 있도록 자율성을 부여합시다.
```suggestion
            dtoList.add(new MainCategoryProductDto(product, thumbnail));
```

Reply:
해당 부분도 학습 후 수정하겠습니다 :exploding_head: